QEfficient/compile/compile_helper.py

```diff
@@ -139,11 +141,12 @@ def compile(
     else:
         custom_io_file_name = "custom_io_fp16.yaml"
 
-    custom_io_file_path = os.path.join(os.path.dirname(onnx_path), custom_io_file_name)
+    if custom_io_file_path is None:
+        custom_io_file_path = os.path.join(os.path.dirname(onnx_path), custom_io_file_name)
 
     if not os.path.isfile(custom_io_file_path):
         raise FileNotFoundError(
-            f"file {custom_io_file_path} needs to exist in the same directory as onnx model files. Please rerun infer/export Api"
+            f"Custom IO file {custom_io_file_name} is neither present at given path nor exists in the onnx model files directory. Please pass the correct file path or rerun infer/export Api"
```

Comment:
f"Custom IO file {custom_io_file_name} is not present at the expected path {custom_io_file_path} nor exists in the onnx model files directory. Please pass the correct file path or rerun infer/export Api"